bundles/org.openhab.binding.groupepsa/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,559 @@
+
+	<channel-type id="mileage">
+		<item-type>Number:Length</item-type>
+		<label>Mileage</label>
+		<description>Total travelled distance</description>
+		<category>Motion</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="position">
+		<item-type>Location</item-type>
+		<label>Position</label>
+		<description>Last known position</description>
+		<category>MoveControl</category>
+		<state pattern="%2$s°N %3$s°E %1$sm"/>
+	</channel-type>
+
+	<channel-type id="heading">
+		<item-type>Number:Angle</item-type>
+		<label>Heading</label>
+		<description>Direction of travel</description>
+		<category>MoveControl</category>
+		<state pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="type">
+		<item-type>String</item-type>
+		<label>Position Type</label>
+		<description>Position acquisition type</description>
+		<category>MoveControl</category>
+		<state readOnly="true">
+			<options>
+				<option value="Estimate">Estimate</option>
+				<option value="Acquire">Acquire</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="signal">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Signal Strength</label>
+		<description>Strength of the position localization signal</description>
+		<category>MoveControl</category>
+		<state pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="lastUpdated">
+		<item-type>DateTime</item-type>
+		<label>Time of Last Update</label>
+		<description>Last time the results were updated on the server</description>
+		<category>Time</category>
+	</channel-type>
+
+	<channel-type id="privacy">
+		<item-type>String</item-type>
+		<label>Privacy status</label>
+		<description>Privacy status</description>
+		<state readOnly="true">
+			<options>
+				<option value="None">None</option>
+				<option value="Geolocation">Location</option>
+				<option value="Full">Full</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="belt">
+		<item-type>String</item-type>
+		<label>Belt status</label>
+		<description>Seat belt status</description>
+		<state readOnly="true">
+			<options>
+				<option value="Normal">Belts on</option>
+				<option value="Omission">Belt open</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="emergency">
+		<item-type>String</item-type>
+		<label>Emergency call</label>
+		<description>Emergency call status</description>
+		<state readOnly="true">
+			<options>
+				<option value="NoRequest">None</option>
+				<option value="AirbagUnabled">Airbag Deployed</option>
+				<option value="Requested">Emergency Call Requested</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="service">
+		<item-type>String</item-type>
+		<label>Service</label>
+		<description>Service Type</description>
+		<state readOnly="true">
+			<options>
+				<option value="Electric">Electric</option>
+				<option value="Hybrid">Hybrid</option>
+				<option value="Unknown">Unknown</option>
```

Comment:
This seems rather a property than a Channel.

Reply:
I'm not entirely certain. It may indeed be, but it could be that it theoretically changes if the car is driving on electricity or on fuel. I propose to keep it a channel just in case.
(There are quite some channels that don't work on all cars, so I can't test it properly.)